dlt/common/storages/live_schema_storage.py

```diff
@@ -55,6 +56,19 @@ def commit_live_schema(self, name: str) -> Schema:
             self._save_schema(live_schema)
         return live_schema
 
+    def is_live_schema_committed(self, name: str) -> bool:
+        """Checks if live schema is present in storage and have same hash"""
+        live_schema = self.live_schemas.get(name)
+        if live_schema is None:
+            raise SchemaNotFoundError(name, f"live-schema://{name}")
+        try:
+            stored_schema_json = self._load_schema_json(name)
+            return live_schema.stored_version_hash == cast(
```

Comment:
we must use `version_hash` here.
stored_version_hash - is a hash of last commit to storage
version_hash - incorporates all the changes
I wonder how the tests are passing. we must somehow bump schema version (which should only happen when saving)